lib/comment-poster.js

```
@@ -164,6 +187,61 @@
+        const mutation = `
+          mutation {
+            resolveReviewThread(input: {threadId: "${comment.id}"}) {
+              thread {
+                isResolved
+              }
+            }
+          }
+        `;
+        
+        execSync(
+          `gh api graphql -f query='${mutation.replace(/'/g, "'")}' --silent`,
```

ESLint

[error] Strings must use singlequote.

(quotes)